src/Raven.Server/Documents/Handlers/BatchHandler.cs

```diff
@@ -21,54 +26,76 @@ public class BatchHandler : DatabaseRequestHandler
         [RavenAction("/databases/*/bulk_docs", "POST")]
         public async Task BulkDocs()
         {
-            DocumentsOperationContext ctx;
-            using (ContextPool.AllocateOperationContext(out ctx))
+            using (ContextPool.AllocateOperationContext(out DocumentsOperationContext ctx))
+            using (var mergedCmd = new MergedBatchCommand{Database = Database})
             {
-                var cmds = await BatchRequestParser.ParseAsync(ctx, RequestBodyStream(), Database.Patcher);
-               
-                var waitForIndexesTimeout = GetTimeSpanQueryString("waitForIndexesTimeout", required: false);
-
-                using (var mergedCmd = new MergedBatchCommand
+                if (HttpContext.Request.ContentType != null) // multipart/mixed
                 {
-                    Database = Database,
-                    ParsedCommands = cmds,
-                })
-                {
-                    if (waitForIndexesTimeout != null)
-                        mergedCmd.ModifiedCollections = new HashSet<string>();
-
-                    try
+                    var boundary = MultipartRequestHelper.GetBoundary(
+                        MediaTypeHeaderValue.Parse(HttpContext.Request.ContentType),
+                        MultipartRequestHelper.MultipartBoundaryLengthLimit);
+                    var reader = new MultipartReader(boundary, RequestBodyStream());
```

Comment:
We need to document that attachments loaded in this manner are limited to 128MB https://github.com/aspnet/HttpAbstractions/blob/dev/src/Microsoft.AspNetCore.Http/Features/FormOptions.cs#L14

And verify that we can upload attachments greater than that we the direct API.